src/utils.js

```diff
@@ -173,7 +173,6 @@ const utils = {
 	/**
 	 * Polyfill a Promise library with missing Bluebird features.
 	 *
-	 * NOT USED & NOT TESTED YET !!!
```

Comment:
Removed this comment as I believe there is a test for it.

Reply:
Thanks, I missed to remove it after I've used in the broker.